e2e/ansible/playbooks/resiliency/test-kubelet-service-stop/test-kubelet-service-stop.yml

```diff
@@ -110,7 +118,7 @@
 
 
        - name: check the node is in NotReady state after kubelet service is stopped
-         shell: source ~/.profile; kubectl get nodes {{node_name}} | grep NotReady | wc -l
+         shell: source ~/.profile; kubectl get nodes {{node_name.stdout}} | grep NotReady | wc -l
          args:
            executable: /bin/bash
          register: result
```

Comment:
you can remove the check for count, keep condition to check if the node is not ready.